content/en/docs/v3.6/learning/api_guarantees.md

```diff
@@ -1,75 +1,60 @@
 ---
-title: KV API guarantees
+title: etcd API guarantees
 weight: 2750
-description: KV API guarantees made by etcd
+description: API guarantees made by etcd
 ---
 
-etcd is a consistent and durable key value store with [mini-transaction][txn]
-support. The key value store is exposed through the KV APIs. etcd tries to
-ensure the strongest consistency and durability guarantees for a distributed
-system. This specification enumerates the KV API guarantees made by etcd.
+etcd is a consistent and durable key value store.
+The key value store is exposed through the KV and Watch [gRPC Services].
+etcd ensures the strongest consistency and durability guarantees for a distributed system.
```

Comment:
This is a little on the nose, as there are stronger consistency models than strict serializability (at last in theory).

Reply:
For now leaving it as it is.